plotting.py

```diff
@@ -309,6 +299,12 @@ def plot_timehistory_at_height(datasets,
         field or height specific colors, limits, etc.
         Example uses include setting linestyle/width, marker, etc.
     """
+    # Avoid FutureWarning concerning the use of an implicitly registered
```

Comment:
Ugh, this is ugly, but I've seen it in other code as well... Is there a reference for where this "fix" comes from?

Reply:
I agree, but leaving a big red warning block inside an example notebook seemed even worse. I basically just took the solution suggested in the FutureWarning message